content/en/docs/setup/install/multicluster/single-network/initial-configuration/index.md

```diff
@@ -0,0 +1,205 @@
+weight: 10
+keywords: [multicluster,first-cluster]
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+To build a {{< gloss >}}multicluster{{< /gloss >}} deployment, configure a
+single cluster {{< gloss >}}service mesh{{< /gloss >}} as your starting point.
+
+The single cluster mesh you deploy in this section, serves as a model for the
+configuration of the first {{< gloss >}}primary cluster{{< /gloss >}}. You need
+at least one primary cluster to complete the [different multicluster deployments.](/docs/ops/deployment/deployment-models/#multiple-clusters)
+
+Each cluster in a multicluster deployment needs a unique identifier. Using
+`CLUSTER_1` as a unique identifier for the first cluster, the following
+diagram shows a service mesh spanning a single primary cluster.
+
+{{< image width="75%"
+    link="init-conf.svg"
+    caption="A service mesh with a single primary cluster"
+    >}}
+
+Complete the [Before you begin instructions](/docs/setup/install/multicluster/#before-you-begin)
+before you continue.
+
+To set up a cluster as the primary cluster of a multicluster service
+mesh, complete the following configuration steps:
+
+1. [Configure trust for the cluster](#configure-trust)
+1. [Deploy Istio to the cluster](#deploy-istio)
+
+## Configure trust
+
+A multicluster service mesh deployment requires that you establish trust between
+the clusters in the mesh. To authenticate workloads across clusters, Istio
+requires that only one root Certificate Authority (CA) is active in the mesh. To
+establish shared trust, generate a single root CA for the entire mesh, and an
+intermediate CA for each cluster. Then, the shared root CA can sign the
+intermediate CA of each cluster.
+
+Production environments should use your organization's
+root CA whenever possible, or set up a secure PKI like [Vault PKI](https://www.vaultproject.io/docs/secrets/pki)
+to generate the common root certificate.
+
+The `Makefile` in [install/certs]({{< github_tree >}}install/certs)
+is a simple tool that helps generate the root certificate and intermediate
+certificates, but do not use it as a production-ready root CA. The tool
+lacks the following critical features:
+
+- Private key protection
+- Access control
+- Auditing
+- Monitoring
+
+Use the `Makefile` to generate the common root and the intermediate certificates
+only for tests or demos. All of the files needed for a cluster CA are stored
+under the subdirectory `<CLUSTER_NAME>`.
+
+Using [the set environment variables](/docs/setup/install/multicluster/#env-var), configure trust with the following steps:
+
+1. Go to the `WORK_DIR` directory with the following command:
+
+    {{< text bash >}}
+    $ cd ${WORK_DIR}
+    {{< /text >}}
+
+1. Generate the root CA and the intermediate CA for the cluster with the
+   following command:
+
+    {{< text bash >}}
+    $ make -f ${ISTIO}/install/certs/Makefile ${CLUSTER_1}-certs
```

Comment:
`make -f ${ISTIO}/tools/certs/Makefile`  1.6 and 1.7 don't have Makefile at this location.

Reply:
I've modified the command following your previous suggestion.

Reply:
same here @nmittler since you own the certs/Makefile you probably know better